checkstyle.xml

```diff
@@ -4,6 +4,10 @@
         "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
 <module name="Checker">
 
+    <module name="SuppressionFilter">
+        <property name="file" value="${config_loc}/checkstyle-suppressions.xml"/>
+    </module>
```

Comment:
I have explained all things in [README](https://github.com/smartcat-labs/smartcat-guidelines/blob/master/coding-guidelines/README.md) on guidelines.